lib/fluent/plugin/out_timestream.rb

```diff
@@ -131,17 +132,18 @@ def create_timestream_records(chunk)
         chunk.each do |time, record|
           dimensions, measure = create_timestream_dimensions_and_measure(record)
           timestream_records.push(create_timestream_record(dimensions, time, measure))
-        rescue MeasureHasEmptyValueError, NoDimensionsError => e
+        rescue EmptyValueError, NoDimensionsError => e
           log.warn("ignore record (#{e})")
+          log.warn("ignored record details: #{record}")
```

Comment:
```suggestion
          log.warn("ignored record due to #{e}")
          log.debug("ignored record details: #{record}")
```

`record` の文字列表示はわかりやすいよね？

Reply:
`record`の内容は以下のようなものになります。
```
2021-03-09 11:40:24 +0900 [debug]: #0 fluent/log.rb:308:debug: ignored record details: {"key1"=>"", "key2"=>nil}
```

読み込んだログファイルの内容に等しいですね。